### ISSUE TITLE
Webcam display covers other elements in consent and config-quality frames

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/ember-lookit-frameplayer#330]
**Bug description**

The webcam video display is covering up text or buttons on the page on certain frame types. 

**Which frame or frames does it affect?**

- exp-lookit-video-consent
- exp-video-config-quality

**To Reproduce**

Create a study with the following protocol
- video consent frame
- video config quality frame:
  - use `"requireTestVideo": true` and `"showRecordMenu": true` settings

**Screenshots**

### Video consent frame

![Screenshot 2023-06-09 at 1 35 52 PM](https://github.com/lookit/ember-lookit-frameplayer/assets/9041788/b63162ac-6e9e-4e07-aac0-a56c641350a3)

### Video config quality frame

![Screenshot 2023-06-09 at 1 35 30 PM](https://github.com/lookit/ember-lookit-frameplayer/assets/9041788/c6ad09f7-df22-4231-a8d9-5a19aaf411dc)
![Screenshot 2023-06-09 at 1 35 03 PM](https://github.com/lookit/ember-lookit-frameplayer/assets/9041788/ca144ede-3333-449a-83bb-f20bc0b679c5)

**Environment (please complete):**

 - Version of frameplayer code (`last_known_player_sha field` on the study edit page): 862604874f7eeff8c9d72adcb8914b21bfb5427e 
 - Reproduced in Chrome 107.0.5304.110, Firefox 109.0.1

**Comment by becky-gilbert** [comment]
This issue might be related to this change on Pipe's end: https://blog.addpipe.com/fully-responsive-recording-client/ And if so, it's probably the interaction with our custom CSS rather than the Pipe changes themselves that is causing the layout problems.

**Comment by mekline** [comment]
@becky-gilbert we had another complaint about this issue on Slack; can you please make sure that QA for Pipe includes checking these frames display correctly with RecordRTC?

**Comment by becky-gilbert** [comment]
@mekline yes absolutely!

**Comment by becky-gilbert** [comment]
Update on this issue for anyone following: 

This webcam display problem was caused by an update to the software that we use for capturing video from participants and displaying the feed/recording back to the user. I spent some time looking into this and wasn't able to find an obvious fix. 

It so happens that we are also in the middle of replacing this software, which will fix this display problem. So given that we are close to rolling out the new video recording method, combined with the non-trivial amount of time it would take to fix this issue with the 3rd party software, we have decided to prioritize our recording method update. After we roll out our update to the recording method, we can take a look at how many researchers are still affected by this (e.g. because they can't or don't want to update their experiment runner version) and then decide if/when to fix it.

Also, I'm looking into whether we can access earlier file versions, before the breaking change, and will update here as I have more information.
